tests/camel-jmh/src/test/java/org/apache/camel/itest/jmh/TypeConverterTest.java

```diff
@@ -87,13 +102,59 @@ public void close() {
             }
         }
 
+        private String getResourceAsString(String resource) {
+            Scanner s = new Scanner(getClass().getClassLoader().getResourceAsStream(resource))
+                    .useDelimiter("\\A");
```

Comment:
Is this a trick to look for a non existing delimiter to let the scanner read the entire file? We have IOHelper class in camel-core you can use to load a file as text, for example the loadText method.